src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/wwwroot/swagger/swagger-utils.js

```diff
@@ -124,28 +149,26 @@ const login = async (swagger, userName, password) => {
         const result = await response.json();
         const accessToken = result.accessToken;
 
-        const obj = {
-            "bearerAuth": {
-                "name": "Bearer",
-                "schema": {
-                    "type": "apiKey",
-                    "description": "JWT Authorization header using the Bearer scheme.",
-                    "name": "Authorization",
-                    "in": "header"
-                },
-                value: accessToken
-            },
-        };
+        const obj = generateAuthorizeObjectForSwagger(accessToken);
```

Comment:
again, please choose a better name for this variable or completely remove it (use its value directly).